src/Native/FactorizationMachineNative/FactorizationMachineCore.cpp

```diff
@@ -0,0 +1,154 @@
+#include <cmath>
+#include <cstring>
+#include <limits>
+#include <pmmintrin.h>
+#include "../Stdafx.h"
+
+#define UNUSED(x) (void)(x)
+#define DEBUG_ONLY(x) (void)(x)
```

Comment:
I don't see any usage of this, can we just remove them?

Reply:
Sure.

---
In reply to: [196942002](https://github.com/dotnet/machinelearning/pull/383#discussion_r196942002) [](ancestors = 196942002)